### PR TITLE
[Feature]: Add missing docstring for dataset

### PR DESCRIPTION
**Pull request by YuanLiuuuuuu**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Please describe the motivation of this PR and the goal you want to achieve through this PR.

## Modification

Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)

Does the modification introduce changes that break the backward compatibility of the downstream repositories?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here and update the documentation.

## Checklist

**Before PR**:

- [ ] Pre-commit or other linting tools are used to fix the potential lint issues.
- [ ] Bug fixes are fully covered by unit tests, the case that causes the bug should be added in the unit tests.
- [ ] The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
- [ ] The documentation has been modified accordingly, like docstring or example tutorials.

**After PR**:

- [ ] If the modification has potential influence on downstream or other related projects, this PR should be tested with those projects, like MMDet or MMSeg.
- [ ] CLA has been signed and all committers have signed the CLA in this PR.

### CODE REVIEW

File: mmselfsup/datasets/transforms/processing.py [hunk 1]
```diff
@@ -468,6 +536,14 @@ def _rotate(self, img: np.ndarray) -> List[np.ndarray]:
         ]
 
     def transform(self, results: dict) -> dict:
+        """Apply rotation augmentation to the given image.
```
Comment (fangyixiao18) [inline mmselfsup/datasets/transforms/processing.py:539]:
```suggestion
        """Apply rotation augmentation to the given image and add ``rot_labels`` to the results.
```

File: mmselfsup/datasets/transforms/processing.py [hunk 2]
```diff
@@ -554,6 +630,14 @@ def _image_to_patches(self, img: np.ndarray) -> List[np.ndarray]:
         return patches, patches_pos
 
     def transform(self, results: dict) -> dict:
+        """Apply random patch augmentation to the given image.
```
Comment (fangyixiao18) [inline mmselfsup/datasets/transforms/processing.py:633]: indicate that the function add patch labels

File: mmselfsup/datasets/transforms/processing.py [hunk 3]
```diff
@@ -853,6 +957,14 @@ def get_params(img: np.ndarray,
         return ymin, xmin, ymax, xmax
 
     def transform(self, results: dict) -> dict:
+        """Randomly crop the image.
```
Comment (fangyixiao18) [inline mmselfsup/datasets/transforms/processing.py]: and resize the image to the target size

**Review by fangyixiao18 — CHANGES_REQUESTED** [review]

File: mmselfsup/datasets/transforms/processing.py (hunk 1 above)
Comment (YuanLiuuuuuu) [reply mmselfsup/datasets/transforms/processing.py:539]: The ``added keys`` already exists in the docstring of this class.

**Review by fangyixiao18 — APPROVED** [review]

**Pull request merged**